libs/e2e-mock/client/src/lib/e2e-mock-client.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { AddKompakt, AddKompaktResponse } from "e2e-mock-server"
+import clientEmiter from "./ipc-client/ipc-client"
+
+export const E2EMockClient = {
+  checkConnection: () => {
+    return !!clientEmiter
+  },
+  addDevice: (kompaktPortInfo?: AddKompakt) => {
```

Comment:
⚪ (suggestion)
Should we introduce validation to initially support only the AddKompakt interface, making it mandatory and not optional?